@here/harp-mapview/lib/DataSource.ts

```diff
@@ -169,6 +169,18 @@ export abstract class DataSource extends THREE.EventDispatcher {
 
     enablePicking: boolean = true;
 
+    /**
+     * Overrides the default rendering order of this `DataSource`.
+     *
+     * @remarks
+     * When `dataSourceOrder` is defined, all the objects created by this `DataSource`
+     * will be rendered on top of the objects created by other `DataSource`s with
+     * lower `dataSourceOrder` values.
+     *
+     * @defaultValue undefined
+     */
+    dataSourceOrder?: number;
```

Comment:
please mark as @beta

Reply:
Align/Remove the WebTileDataSource renderOrder Parameter with this change

Reply:
``WebTileDataSourceOptions.renderingOptions.renderOrder`` should be deprecated/removed in favour of common ``dataSourceOrder`` option.

Reply:
@germanz , I have done it in a follow up commit